tools/gfx/render.cpp

```diff
@@ -90,6 +90,12 @@ extern "C"
                 return SLANG_FAIL;
             }
             break;
+#elif SLANG_LINUX_FAMILY
+            case DeviceType::Default:
+            case DeviceType::Vulkan:
+            {
+                return createVKDevice(desc, outDevice);
+            }
```

Comment:
Shouldn't we also support creating a CUDA device on Linux? That option seems easy enough to support, right?

Reply:
You are right.